pkg/globalnet/controllers/cluster_egressip_controller.go

```diff
@@ -128,14 +145,177 @@ func (c *clusterGlobalEgressIPController) onCreate(egressIP *submarinerv1.Cluste
 				ClusterGlobalEgressIPName),
 		})
 
+		return errors.Errorf("ClusterGlobalEgressIP with name %q is not supported, only well-known"+
+			" name %q is supported", egressIP.Name, ClusterGlobalEgressIPName)
+	}
+
+	if *numberOfIPs < 0 {
+		tryAppendStatusCondition(&egressIP.Status, &metav1.Condition{
+			Type:    string(submarinerv1.GlobalEgressIPAllocated),
+			Status:  metav1.ConditionFalse,
+			Reason:  "InvalidInput",
```

Comment:
We should make those Reason values part of the submarinerv1 module, as constants, so then the users of the API can compare to those constant values when consuming the API.

Reply:
Yeah, makes sense. WDYS @tpantelis

Reply:
yeah we can although not sure what other users of our Go API we would have.  The Reason is really a string for end-users.